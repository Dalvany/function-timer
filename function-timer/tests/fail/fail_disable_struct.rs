use function_timer::time;

struct Test {}

#[time(disable)]
impl Test {
    fn test() {}
}
