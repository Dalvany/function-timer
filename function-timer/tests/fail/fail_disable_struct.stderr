error: You can't disable a whole impl block
 --> tests/fail/fail_disable_struct.rs:5:8
  |
5 | #[time(disable)]
  |        ^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fail/fail_disable_struct.rs:8:2
  |
8 | }
  |  ^ consider adding a `main` function to `$DIR/tests/fail/fail_disable_struct.rs`
